VisualStudio.gitignore

```diff
@@ -1,16 +1,18 @@
-## Ignore Visual Studio temporary files, build results, and
-## files generated by popular Visual Studio add-ons.
-
-# User-specific files
+# temporary files generated by Visual Studio specific to the local user
```

Comment:
This is an example of gathering up rules into a group. I'll grammarize this later, but the TL;DR is:

 - context to explain the group
 - before each group of rules, you could specify a `url` or `remove` group
 - newline comment between sections for readability